gprofiler/metadata/application_identifiers.py

```diff
@@ -273,6 +273,21 @@ def get_app_id(self, process: Process) -> Optional[str]:
         return None
 
 
+class _NodeModuleApplicationIdentifier(_ApplicationIdentifier):
+    def get_app_id(self, process: Process) -> Optional[str]:
+        skip = False
+        for arg in process.cmdline()[1:]:
+            if skip:
+                skip = False
+                continue
+            if arg in ["--require", "-r"]:
```

Comment:
`node --require=b.js r.js` is also a legit commandline, let's also check `if arg.startswith("--require=")`

Reply:
I don't see this change committed